libwasmvm/bindings.h

```diff
@@ -260,6 +260,16 @@ typedef struct Iterator_vtable {
                      struct UnmanagedVector*,
                      struct UnmanagedVector*,
                      struct UnmanagedVector*);
+  int32_t (*next_key_db)(struct iterator_t,
+                         struct gas_meter_t*,
+                         uint64_t*,
+                         struct UnmanagedVector*,
+                         struct UnmanagedVector*);
+  int32_t (*next_value_db)(struct iterator_t,
+                           struct gas_meter_t*,
+                           uint64_t*,
+                           struct UnmanagedVector*,
+                           struct UnmanagedVector*);
```

Comment:
This `_db` suffix has always been redundant here I think, especially when namespaces inside the `Iterator_vtable`. I think you can just drop them and then we rename `next_db` -> `next`.